src/AppBanner/index.tsx

```diff
@@ -57,39 +44,25 @@ export const AppBanner = (props: Props): ReactElement | null => {
 
     const [showBanner, setShowBanner] = useState(true);
 
-    const onCloseClick = (
-        evt: React.MouseEvent<HTMLButtonElement, MouseEvent>,
-        internalButtonId: number,
-    ): void => {
-        evt.preventDefault();
+    if (!canRender(props.brazeMessageProps)) {
+        return null;
+    }
 
+    const onCloseClick: OnCloseClick = (evt, internalButtonId) => {
```

Comment:
This will go away when I refactor banners for accessibility. In the meantime, TS seems happy that it can infer the type from elsewhere so I suggest we leave it?